xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -61,9 +62,9 @@ CVideoPlayerAudio::CVideoPlayerAudio(CDVDClock* pClock, CDVDMessageQueue& parent
   m_prevskipped = false;
   m_maxspeedadjust = 0.0;
 
-  // 18 MB allows max bitrate of 18 Mbit/s (TrueHD max peak) during 8 seconds
+  // 18 MB allows max bitrate of 18 Mbit/s (TrueHD max peak) during m_messageQueueTimeSize seconds
```

Comment:
Comment only reflects the worst case and this is still 8 seconds. I mean SetMaxDataSize is still initialized to 18 *  1024 * 1024 and not based in current `m_messageQueueTimeSize` then maybe keep code comment with 8 second or change code to initialize proportionally to `m_messageQueueTimeSize`